### DISCUSSION TITLE
Can't get Frigate running no more

### DISCUSSION THREAD

**Discussion by Tomiol80** [blakeblackshear/frigate#3486] [General]
At the fist I got everything running smoothly, but now I cannot get it run even on smallest config.

Any advice?

frigate    | [services.d] starting services
frigate    | [services.d] done.
frigate    | [2022-07-17 23:19:43] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
frigate    | Starting migrations
frigate    | [2022-07-17 23:19:43] peewee_migrate                 INFO    : Starting migrations
frigate    | There is nothing to migrate
frigate    | [2022-07-17 23:19:43] peewee_migrate                 INFO    : There is nothing to migrate
frigate    | [2022-07-17 23:19:43] detector.cpu1                  INFO    : Starting detection process: 218
frigate    | [2022-07-17 23:19:43] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
frigate    | [2022-07-17 23:19:43] detector.cpu2                  INFO    : Starting detection process: 220
frigate    | [2022-07-17 23:19:43] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
frigate    | [2022-07-17 23:19:43] detector.cpu3                  INFO    : Starting detection process: 222
frigate    | [2022-07-17 23:19:43] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Output process started: 226
frigate    | [2022-07-17 23:19:44] detector.cpu4                  INFO    : Starting detection process: 223
frigate    | [2022-07-17 23:19:44] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Camera processor started for etupiha: 228
frigate    | [2022-07-17 23:19:44] ws4py                          INFO    : Using epoll
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Camera processor started for takapiha: 231
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Camera processor started for tallinpiha: 233
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Camera processor started for ponientarha: 235
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Camera processor started for tallikamppa: 236
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Camera processor started for tallikaytava: 237
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Capture process started for etupiha: 238
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Capture process started for takapiha: 241
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Capture process started for tallinpiha: 244
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Capture process started for ponientarha: 246
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Capture process started for tallikamppa: 251
frigate    | [2022-07-17 23:19:44] frigate.app                    INFO    : Capture process started for tallikaytava: 255
frigate    | [2022-07-17 23:19:44] ws4py                          INFO    : Using epoll
frigate    | [2022-07-17 23:19:44] frigate.mqtt                   INFO    : Turning off recordings for tallikamppa via mqtt
frigate    | [cont-finish.d] executing container finish scripts...
frigate    | [cont-finish.d] done.
frigate    | [s6-finish] waiting for services.
frigate    | [s6-finish] sending all processes the TERM signal.
frigate    | [s6-finish] sending all processes the KILL signal and exiting.
frigate exited with code 0

frigate.yml
mqtt:
  host: 192.168.1.7

detectors:
  cpu1:
    type: cpu
    num_threads: 3
  cpu2:
    type: cpu
    num_threads: 3
  cpu3:
    type: cpu
    num_threads: 3
  cpu4:
    type: cpu
    num_threads: 3

ffmpeg:
  hwaccel_args:
    - -hwaccel
    - cuda
    - -hwaccel_device
    - /dev/dri/card0
    - -hwaccel_output_format
    - cuda

cameras:
  etupiha:
    ffmpeg:
      inputs:
        - path: rtsp://xxx:xxx@192.168.1.41:554/Streaming/Channels/101
          roles:
            - detect
            - rtmp
  takapiha:
    ffmpeg:
      inputs:
        - path: rtsp://xxx:xxx@192.168.1.42:554/Streaming/Channels/101
          roles:
            - detect
            - rtmp
  tallinpiha:
    ffmpeg:
      inputs:
        - path: rtsp://xxx:xxx@192.168.1.43:554/Streaming/Channels/101
          roles:
            - detect
            - rtmp
  ponientarha:
    ffmpeg:
      inputs:
        - path: rtsp://xxx:xxx@192.168.1.44:554/Streaming/Channels/101
          roles:
            - detect
            - rtmp
  tallikamppa:
    motion:
      mask:
        - 1280,635,1229,187,802,0,0,0,0,720,727,720
    ffmpeg:
      inputs:
        - path: rtsp://xxx:xxx@192.168.1.45:554/stream1
          roles:
            - detect
            - rtmp

  tallikaytava:
    ffmpeg:
      inputs:
        - path: rtsp://xxx:xxx@192.168.1.46:554/stream1
          roles:
            - detect
            - rtmp

record:
  enabled: True
  expire_interval: 60
  retain:
    days: 7
    mode: motion
  events:
    max_seconds: 300
    pre_capture: 5
    post_capture: 5
    objects:
      - person
    required_zones: []
    retain:
      default: 10
      mode: motion
      objects:
        person: 15
live:
  height: 720
  quality: 8

docker-compose.yml
`version: '3.9'
services:
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: blakeblackshear/frigate:stable-amd64nvidia
    runtime: nvidia
    environment:
      - NVIDIA_VISIBLE_DEVICES=all
      - NVIDIA_DRIVER_CAPABILITIES=compute,utility,video
    devices:
      - /dev/bus/usb:/dev/bus/usb
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
      - /dev/dri/card0
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home/tomi/.config/frigate.yml:/config/config.yml:ro
      - /mnt/raid:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "1935:1935"   # RTMP feeds

**Comment by NickM-27** [comment]
So `192.168.1.7` is still your MQTT host correct? (you can use MQTT explorer to view it, this will help confirm). Also the broker does not have username or password setup?

**Reply by Tomiol80** [reply]
Yes. I can see frigate connection on iobroker. And no user/password is required


mqtt.0 | 2022-07-17 23:35:56.292 | info | Client [frigate] connection closed: closed
mqtt.0 | 2022-07-17 23:35:55.979 | info | Client [frigate] publishOnSubscribe send all known states
mqtt.0 | 2022-07-17 23:35:55.778 | info | Client [frigate] subscribes on "frigate/#" with regex /^mqtt\.0\.frigate(\..*)?/
mqtt.0 | 2022-07-17 23:35:55.778 | info | Client [frigate] subscribes on "frigate/#" with regex /^frigate(\..*)?/
mqtt.0 | 2022-07-17 23:35:55.777 | info

**Reply by Tomiol80** [reply]
You was rigth. There was something wrong with MQTT server. I reinstalled it and got the frigate running again!

**Reply by NickM-27** [reply]
Cool, glad it is working again
